app/templates/drupal/lightning/8.x/composer.json

```diff
@@ -0,0 +1,17 @@
+{
+    "name": "client/project",
+    "description": "{Project} drupal codebase for {client}.",
+    "require-dev": {
+        "behat/mink-zombie-driver": "~1.2",
+        "drupal/drupal-extension": "~3.0",
+        "drush/drush": "dev-master",
+        "phpmd/phpmd": "~2.1",
+        "drupal/coder": "^8.2",
+        "guzzlehttp/guzzle" : "^6.0@dev",
+        "symfony/dependency-injection": "2.7.*",
+        "symfony/event-dispatcher": "2.7.*"
+    },
+    "require": {
+        "roave/security-advisories": "dev-master"
+    }
+}
```

Comment:
This composer.json does not seem different from the generic D8 composer.json. Furthermore, the changes in #66 will cause this to be deprecated. Whichever one goes in first will block the other.